src/orca/services/synapse/ops.py

```diff
@@ -57,3 +61,101 @@ def monitor_evaluation_queue(self, evaluation_id: str) -> bool:
         )
         submissions_num = sum(1 for submission in received_submissions)
         return submissions_num > 0
+
+    def trigger_indexing(self, synapse_view: str) -> None:
+        """
+        Trigger indexing in Synapse.
+
+        Arguments:
+            syn: A Synapse object.
+            synapse_view: The Synapse ID of the view.
+        """
+        self.client.tableQuery(f"select * from {synapse_view} limit 1")
+
+    def is_valid_submission_status(self, submission_status: str) -> bool:
+        """
+        Check if submission status is valid. Based on:
+        https://rest-docs.synapse.org/rest/org/sagebionetworks/evaluation/model/SubmissionStatusEnum.html
+
+        Arguments:
+            submission_status: The submission status to check.
+
+        Returns:
+            Boolean indicating if submission status is valid.
+        """
+
+        # List all the available status options based on API docs
+        status_options = [
```

Comment:
I don't love that we are basically re-implementing the Synapse API enum here. I also am not sure this function is necessary at all considering `get_submissions_with_status` is a table query function. You can query the table with any string you want, and if that string is incorrect you should receive no responses since that is what the query would return. Do you feel strongly that this needs to be included?

Reply:
If you do think this should remain as part of this work, I think we should implement this as an [enum](https://docs.python.org/3/library/enum.html) of our own. 

In the future it should probably be added to the `synapseclient` as part of a `Submissions` OOP model.

Reply:
I had the same frame of mind - my original plan was to pull the available statuses from the API but I found no way of doing that, so I went with recreating the list, but it does make this a little flimsy.

I don't feel strongly about keeping this function, but I do feel strongly about not letting a typo in the status argument get passed along quietly. At the very least, I think it would be good to add a warning in `get_submissions_with_status` like `warnings.warn("This query returned an empty list. Ensure ``submission_status`` is correct")` to help catch potential mistakes. What do you think of this compromise?

Reply:
Fair enough - In that case, let's not handle this just yet: I'll make a ticket to create an Enum for this in the python client, and one for revisiting this type of input error after the Enum gets created. I'm going to revert these changes in the next commit.